src/utils.ts

```diff
@@ -50,3 +50,7 @@ export let to_lsp_position = (pos: IPosition): Position => {
 export let get_range = (ps: IPositionedString): Range => {
   return _get_range_che(ps);
 };
+
+export let get_golang_executable = (): string => {
```

Comment:
Since it is part of the config, I would recommend to add this part of [AnalsisConfig](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/blob/master/src/server.ts#L134) structure. WDYT?

Reply:
Initial I thought of this class, but we need to path in server.js, collector.js and test code. Where as AnalysisConfig is local to server.js. Should I export this class from server.js / move it to a separate file.

Reply:
Refactoring of moving configuration into separate file and class is done.